src/Config/Validator/Deploy/EnvironmentVariables.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Config\Validator\Deploy;
+
+use Magento\MagentoCloud\App\Error;
+use Magento\MagentoCloud\Config\Environment;
+use Magento\MagentoCloud\Config\Validator;
+use Magento\MagentoCloud\Config\ValidatorException;
+use Magento\MagentoCloud\Config\ValidatorInterface;
+use Magento\MagentoCloud\Package\MagentoVersion;
+
+/**
+ * Validates that environment variables doesn't contain redundant variables.
```

Comment:
```suggestion
 * Validates that environment variables does not contain invalid variables.
```

Reply:
I'd suggest that it should be "do not" - better English.